spec/index.bs

```diff
@@ -10845,6 +10843,72 @@ It must only be included by interfaces which also include those mixins.
         </div>
 </dl>
 
+<div class=note heading id=dynamicOffsetOrder>
+    Dynamic offset are applied in {{GPUBindGroupLayoutEntry}}.{{GPUBindGroupLayoutEntry/binding}} order.
+
+    This means that if `dynamic bindings` is the list of each {{GPUBindGroupLayoutEntry}} in the {{GPUBindGroupLayout}}
+    with {{GPUBindGroupLayoutEntry/buffer}}?.{{GPUBufferBindingLayout/hasDynamicOffset}} set to `true`, sorted by
+    {{GPUBindGroupLayoutEntry}}.{{GPUBindGroupLayoutEntry/binding}}, then `dynamic offset[i]`, as supplied to
+    [=GPUBindingCommandsMixin/setBindGroup()=], will correspond to `dynamic bindings[i]`.
+
+    <div class=example>
+        For a {{GPUBindGroupLayout}} created with the following call:
+
+        <pre highlight=js>
+            // Note the order of the binding properties!
```

Comment:
Maybe more text here?
```suggestion
            // Note the bindings are listed out-of-order in this array, but it
            // doesn't matter because they will be sorted by binding index.
```